spec/classes/server_spec.rb

```diff
@@ -31,7 +31,7 @@
             :servertype   => server_type,
             :storeconfigs => 'puppetdb',
             :environmentpath => '$confdir/environments',
-            :basemodulepath => '/etc/puppet/modules:/usr/share/puppet/modules',
+            :basemodulepath => ['/etc/puppet/modules', '/usr/share/puppet/modules'],
```

Comment:
@danieldreier i'm **not** saying **requiring** puppet 4

i'm said that it's really easy to keep this backwards compatible, while also being future proof.